docs/faq/faq-questions.md

```diff
@@ -1,3 +1,66 @@
-# Frequently Asked Questions
+## General
 
-[Can I deleted a product from my UpdateHub?](faq/answers/answers.md)
+### What is **UpdateHub**?
+**UpdateHub** is a tool that turns all of the firmware process easy and secure. It runs all tasks involved in the updating execution and allows the user to set the behavior of activity flow. The **UpdateHub** also allows the user to check and manage his device’s fleet in a unique location.
+
+### Which versions are available?
+**UpdateHub** has two flavors: *Open Source* and *Cloud*. The tool is made of two parts: an agent (internal in devices) and a server. The **UpdateHub Agent** is the same in both versions, but the server on the *Open Source* version is hosted by the user, while **UpdateHub Cloud** is hosted in a dedicated, secure cloud server.
+
+### The use of **UpdateHub** is free?
+The *Open Source* version is totally free forever. The *Cloud* version allows registering until 5 devices without pay.
```

Comment:
... The *Cloud* version allows registering up to 5 free devices.

Reply:
@otavio done